molo/core/models.py

```diff
@@ -31,13 +33,45 @@ class HomePage(Page):
         help_text=_('Optional page to which the banner will link to')
     )
 
+    COMMENTING_STATE_CHOICES = (
+        ('O', 'Open'),
+        ('C', 'Closed'),
+        ('D', 'Disabled'),
+        ('T', 'Timestamped'),
+    )
+    commenting_state = models.CharField(max_length=1,
+                                        choices=COMMENTING_STATE_CHOICES,
+                                        blank=True,
+                                        null=True)
+    commenting_open_time = models.DateTimeField(null=True, blank=True)
+    commenting_close_time = models.DateTimeField(null=True, blank=True)
+
     parent_page_types = ['core.LanguagePage']
     subpage_types = ['core.ArticlePage']
 
+    def get_effective_commenting_settings(self):
+        # return commenting settings for the homepage
+        if (self.commenting_state and self.commenting_state != ""):
```

Comment:
`""` always resolves to `False` so I think you can just do `if self.commenting_state:` here instead.